packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -521,6 +521,67 @@ const DatePickerInput = (props) => {
     }
   };
 
+
+
+  const handleInputKeydown = (event, inputType) => {
+    const { day, month, year } = date;
+    let inputDate;
+    let formattedDate;
+    if (day.length === 2 && month.length === 2 && year.length === 4) {
+      inputDate = DateUtil.convertToISO8601(`${year}-${month}-${day}`, DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      formattedDate = DateUtil.strictFormatISODate(inputDate, momentDateFormat);
```

Comment:
What is the difference between this step and the preceding line?

Reply:
format, 
`inputDate` = YYYY-MM-DD
`formatted` = locale format that is being displayed (so in my case, for example MM/DD/YYYY)